src/Common/src/System/Net/CookieParser.cs

```diff
@@ -534,11 +534,7 @@ private static bool InternalSetNameMethod(Cookie cookie, string value)
                     // Cookie.set_Name keeps the original name if the string is empty or null.
                     // Unfortunately this API is internal so we use reflection to access it. The method is cached for performance reasons.
                     BindingFlags flags = BindingFlags.Instance;
-#if uap
-                    flags |= BindingFlags.Public;
-#else
                     flags |= BindingFlags.NonPublic;
```

Comment:
Seems like this can now be:
```C#
const BindingFlags Flags = BindingFlags.Instance | BindingFlags.NonPublic;
```
or just pass `BindingFlags.Instance | BindingFlags.NonPublic` directly as the argument.